resources/ts/components/DegreeProgramEditForm/General.tsx

```diff
@@ -432,7 +432,7 @@ const General = () => {
 						fill="full"
```

Comment:
Does it make sense to make the layout half/half with previous videos field?

Reply:
It does make sense, but it is out of scope. 😉